docker-compose.yml

```diff
@@ -1,20 +1,23 @@
 version: "3.1"
+
 services:
+
   proxy:
     image: vfarcic/docker-flow-proxy
     ports:
       - 80:80
       - 443:443
     networks:
-      - default
+      - proxy
```

Comment:
It's better to use an externally defined network since the final name will not be prefixed with the name of the stack. If it's `default` and defined as internal, the name will be `[STACK_NAME]_default` and cause confusion when defining the same network in other stacks.